ui/app/templates/components/oidc/assignment-form.hbs

```diff
@@ -0,0 +1,82 @@
+<div ...attributes>
+  <PageHeader as |p|>
+    <p.top>
+      <nav class="breadcrumb" aria-label="breadcrumbs">
+        <ul>
+          <li>
+            <span class="sep">&#x0002f;</span>
+            {{#if @model.isNew}}
+              <LinkTo @route={{"vault.cluster.access.oidc.assignments"}}>
+                Assignments
+              </LinkTo>
+            {{else}}
+              {{! You're editing in this view }}
+              <LinkTo @route={{"vault.cluster.access.oidc.assignments.assignment.details"}} @model={{@model.name}}>
+                Details
+              </LinkTo>
+            {{/if}}
+          </li>
+        </ul>
+      </nav>
+    </p.top>
+    <p.levelLeft>
+      <h1 class="title is-3" data-test-oidc-assignment-title>
+        {{if @model.isNew "Create" "Edit"}}
+        assignment
+      </h1>
+    </p.levelLeft>
+  </PageHeader>
+
+  <form {{on "submit" (perform this.save)}}>
+    <div class="box is-sideless is-fullwidth is-marginless">
+      <FormFieldLabel for="name" @label="Name" />
+      <input
+        autocomplete="off"
+        spellcheck="false"
+        value={{@model.name}}
+        disabled={{not @model.isNew}}
+        class="input field {{if this.errors.name.errors 'has-error-border'}}"
+        {{on "input" this.handleOperation}}
+      />
+      {{#if this.errors.name.errors}}
+        <AlertInline @type="danger" @message={{join ", " this.errors.name.errors}} />
```

Comment:
Does this mean the errors will be joined with a `,` _and_ a `.` because of the join here and in the save task?

Reply:
I changed this so it doesn't show the API errors here. Only the validation errors. If there's an API error, then it will show as a flashMessage, the same pattern for the scopes-form.